internal/signerverifier/ssh/ssh.go

```diff
@@ -101,14 +104,22 @@ func NewKeyFromFile(path string) (*sv.SSLibKey, error) {
 		return nil, fmt.Errorf("failed to parse SSH2 key: %w", err)
 	}
 
-	return &sv.SSLibKey{
+	key := &sv.SSLibKey{
 		KeyID:   ssh.FingerprintSHA256(sshPub),
 		KeyType: SSHKeyType,
 		Scheme:  sshPub.Type(),
 		KeyVal: sv.KeyVal{
 			Public: base64.StdEncoding.EncodeToString(sshPub.Marshal()),
 		},
-	}, nil
+	}
+
+	keyID, err := calculateKeyID(key)
```

Comment:
I made this change first, but we may not need it with the addition of MetadataKey(). I'm still not sure the best way to handle key IDs, we should discuss @lukpueh

Reply:
Why should it be necessary?

You can take an ssh key with arbitrary keyid, create a verifier with `NewVerifierFromKey`, and verify a matching signature. So, if you are worried about verifying existing metadata, the keyid format should not be a problem.

What would be a problem is,  if `NewKeyFromFile` must compute the same default keyid as previous function that loaded the same key from file. But I don't think that's a necessary requirement.

Reply:
Yeah, especially since our key IDs will change _regardless_ for the same ssh key we may have used before. I've dropped this and gone back to ssh fingerprint. It's also nice that we use the ssh fingerprint here, gpg fingerprint for gpg etc, this way.